torch/csrc/jit/codegen/cuda/tensor_view.cpp

```diff
@@ -1289,6 +1290,8 @@ TensorViewBuilder& TensorViewBuilder::shape(const std::vector<int64_t>& shape) {
   for (int64_t i : shape) {
     if (i == -1) {
       shape_.emplace_back(IrBuilder::create<Int>());
+    } else if (i == 1) {
+      shape_.emplace_back(FusionGuard::getCurFusion()->oneVal());
```

Comment:
Is there any specific reason to use the one val? Do we also want to use the zero val?

Reply:
Just trying to save an instance creation; I don't think it will make much difference here. And yes, we should also use zero val, because why not.